src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -282,135 +313,64 @@ public string GetRelativePath(string filePath)
         }
 
         /// <summary>
-        /// Enumerate all the PowerShell (ps1, psm1, psd1) files in the workspace in a recursive manner
+        /// Whether the current runtime is Dot Net Framework as opposed to Core or Native
         /// </summary>
-        /// <returns>An enumerator over the PowerShell files found in the workspace</returns>
-        public IEnumerable<string> EnumeratePSFiles()
+        private bool IsDotNetFrameWork()
```

Comment:
We use this same comparison in the PowerShellContext constructor. Eventually we should create a new static class in the Utility directory with a cached result in it.

Reply:
Done.  Now using the new util method from https://github.com/PowerShell/PowerShellEditorServices/pull/911